cannon/mipsevm/exec/calling_convention.go

```diff
@@ -0,0 +1,56 @@
+package exec
```

Comment:
A few pieces of overall feedback:

- I would only add constants that we are actually using.
- Whatever changes we make in the go code should be mirrored in solidity (adding constants there and replacing their usage).

Reply:
+1 on this.

Reply:
> I would only add constants that we are actually using.

What about just commenting out the unused ones in golang, so that in future, we can uncomment them if we need?

Reply:
We should avoid introducing unnecessary commentary that doesn't serve the VM. Let's simply remove the unused constants.

Reply:
OK, removed unused ones.